src/autoloads/win-loss-flag.gd

```diff
@@ -0,0 +1,18 @@
+extends Node
+
+var WIN = true;
+var LOSS = false;
```

Comment:
Either use an actual enum and pass enum values, or only use booleans.

Reply:
Updated 👍

Reply:
Updated to use an enum for the flag conditions